README.md

```diff
@@ -94,6 +108,34 @@ or
 bundle exec scss-lint app/webpacker/styles
 ```
 
+Vscode - Rubocop has a vscode extension, linting may need to be turned on
+
+## Using Amazon S3 Bucket for asset storage (e.g. in production)
+
+Ensure you have added 4 parameters to the .env file, these are (ACCESS_KEY_ID, SECRET_ACCESS_KEY, REGION, BUCKET) e.g:
+
+```
+ACCESS_KEY_ID=7867867687
+SECRET_ACCESS_KEY=876876876
+REGION=eu-west-2
+BUCKET=eyfsreformspike
```

Comment:
Are these actual access keys and secrets? If so, they will need to be purged

Reply:
no they are just random numbers